.github/STYLEGUIDE.md

```diff
@@ -743,6 +743,37 @@ Each guide is broken into a few sections. Sections contain a list of guidelines.
   ```
   </details>
 
+* **PREFER attributes on the same line as what they are attributing.**
+
+  <details>
+
+  #### Why?
+
+    Because we naturally read lines of code as a consistent line, introducing whitespace makes it that much harder to understand the full context of the line you're trying to understand. This is the same reasoning we put behind preferring a return on the same line as the final parameter in a function declaration.
+
+    NOTE: This does not come at the cost of sanity. If you have an exceptionally large number of attributions then it might indicate that you've gotten overzealous, and it actually obscures your meaning, rather than add to it.
+
+  ```swift
+  // WRONG
+  @objc
+  func thingExposedToObjectiveC() {
+    // ...
+  }
+  
+  // WRONG
+  @ViewBuilder @discardableResult @MainActor @inlinable func betYouDoNotEvenKnowWhatThisFunctionDoesAnymore() {
```

Comment:
This one should be clearer that what we mean is "You're doing something wrong if you have this many attributes" rather than "Split them up on separate lines"

Reply:
Part of me wants to say that many attributes on a function or property are simply a smell and since this is a "PREFER" I'm tempted to say don't worry about it?

That being said I am not at all a fan of 1 attribute per line, I much prefer the logical breakdown of related attributes, if possible.

Reply:
I think I'm looking for an example of what is something that is readable but still long.  And maybe a why...

```swift
@DependencyInjected @objc @CustomWorkflowAttribute var variableName: String?
```

Reply:
Added an example with 2 attributes on the same line.